GameData/RealismOverhaul/Engine_Configs/Veronique_Config.cfg

```diff
@@ -310,70 +252,53 @@
 
 	RESOURCE
 	{
-		name = Tonka250
+		name = Furfuryl
 		amount = 1
 		maxAmount = 1
 	}
 }
 
-@PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[VeroniqueR]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
-{
-	TESTFLIGHT
-	{
-		//7 successes, 1 failure
-		name = VeroniqueR
-		ratedBurnTime = 6.5
-		ignitionReliabilityStart = 0.85
-		ignitionReliabilityEnd = 0.97
-		cycleReliabilityStart = 0.875000
-		cycleReliabilityEnd = 0.975000
-
-	}
-
-	@MODULE[ModuleEngineConfigs] { @CONFIG[VeroniqueR] { %ratedBurnTime = #$/TESTFLIGHT[VeroniqueR]/ratedBurnTime$ } }
-}
 @PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[Veronique]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
 {
 	TESTFLIGHT
 	{
-		//35 successes, 19 failures
+		//Awful reliability, poor sample size. Just make up numbers
 		name = Veronique
 		ratedBurnTime = 45
-		ignitionReliabilityStart = 0.85
-		ignitionReliabilityEnd = 0.97
-		cycleReliabilityStart = 0.648148
-		cycleReliabilityEnd = 0.929630
-		techTransfer = VeroniqueR:50
+		ignitionReliabilityStart = 0.75
+		ignitionReliabilityEnd = 0.95
+		cycleReliabilityStart = 0.75
+		cycleReliabilityEnd = 0.95
 	}
 	@MODULE[ModuleEngineConfigs] { @CONFIG[Veronique] { %ratedBurnTime = #$/TESTFLIGHT[Veronique]/ratedBurnTime$ } }
 }
 @PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[VeroniqueAGI]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
 {
 	TESTFLIGHT
 	{
-		//8 successes, 1 failure
+		//Awful reliability, poor sample size. Just make up numbers
```

Comment:
That one is half-decent, contraly to the non-AGI model above:
out of 48 AGI launches, 81.5% were successful (Huwart 2004, p. 147). Unsure how much of that is engine failures as opposed to other failures.